<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main</class>
 <widget class="QWidget" name="main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Main</string>
  </property>
  <widget class="QLabel" name="label_friends">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>好友列表</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_friends">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>171</width>
     <height>321</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>2017011626 在线</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2017011542 在线</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2017011547 在线</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3017011626 在线</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3017011542 在线</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3017011547 在线</string>
    </property>
   </item>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_msg">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>341</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_input">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>310</y>
     <width>341</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_send">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>420</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_addFriend">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>450</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确认添加</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_addFriend">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>410</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_addFriend">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>添加好友：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_updateState">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
